dev/io.openliberty.mcp.internal_fat/publish/servers/mcp-server/server.xml

```diff
@@ -8,6 +8,6 @@
   </featureManager>
 
   <include location="../fatTestPorts.xml" />
-  <logging traceSpecification="MCP=all"/>
+  <!--logging traceSpecification="MCP=all"/-->
```

Comment:
Could you delete this line instead of commenting it out please?

We don't actually need it in both the server.xml and the bootstrap.properties anyway.